utils/setupcalls.py

```diff
@@ -0,0 +1,13 @@
+import subprocess, os, sys, platform
+
+
+class SetupCalls():
+
+    def SetUpLogging():
+        print(f"\r\nSetting up file logging...")
+        tee = subprocess.Popen(["tee", "log_LatestRun.txt"], stdin=subprocess.PIPE)
```

Comment:
Is there a reason we need this instead of just using the `logging` module?

Reply:
I did this for pytest originally to put the log in a file for debugging, but the locust log is not useful for that. It's not removed yet just like the other pytest code. I was planning to keep it until the test development as a fallback case if needed.